src/components/pages/DatasetsList.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import ReactRouterPropTypes from 'utils/react-router-prop-types';
+import PropTypes from 'prop-types';
+import { Search, Eye } from 'react-feather';
+
+const DatasetItem = (props) => {
+  const {
+    name, description,
+  } = props;
```

Comment:
do it in one line as there are few props

Reply:
fixed;

Reply:
push to repo, if fixed.